build/configs/rtl8721csm/security_hal_test_tz/Make.defs

```diff
@@ -0,0 +1,154 @@
+###########################################################################
```

Comment:
Can we use rdp config only? the config name suggest its for test case

Reply:
If use rdp config need to enable Ameba SE and security hal test. Do you mean this?

Reply:
I was suggesting to use only one config for Trustzone changes, but anyway now TZ is enabled in loadable.
May be later SE can be added to loadable config.